Dockerfile

```diff
@@ -12,6 +12,6 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get -y update && apt-get -y upgrade && apt-get -y install git curl python python-requests supervisor
 RUN curl https://bootstrap.pypa.io/get-pip.py | python
 ADD resources/supervisor/supervisord.conf /etc/supervisor/supervisord.conf
-RUN pip install git+https://github.com/autodesk-cloud/ochopod.git
+RUN pip install git+https://github.com/gomezsan/ochopod.git
```

Comment:
Right, fair enough. I guess that the reason why I liked the embedded SDK is because testing Ochopod in local mode on Windows is currently not really possible (or at least I had issues with that: log file location is hardcoded to /var/log/ochopod.log, environment variables issues,...), which forces me to generate a docker image every time I want to test something. Really not fun nor productive...